src/Uno.Extensions.Templates/content/unoapp-extensions/azure-pipelines.yml

```diff
@@ -0,0 +1,217 @@
+trigger:
+- master
+- main
+- release
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  #[REQUIRED]
+  #For connecting your azureSubscription to the pipeline:
+  #1.- Go to your project settings (Bottom Left option)
+  #2.- Scroll down to 'Service Connections'
+  #3.- Click on 'Create new service connection' button
+  #4.- Click on 'Azure Resource Manager' and ckick on Next
+  #5.- Select ''Service Principal (automatic) and click on Next
+  #6.- Select the subscription you want to use and set a Name for your new service connection
+  #7.- Finally, use the name you defined in previous step as value for this variable
+- name: azureAccount
+  value: '$azureSubscription$' 
+
+- name: jobTimeout
+  value: '40'
+
+stages:
+  - stage: Build_UNO
+    pool:
+      vmImage: 'windows-latest'
+
+    jobs:
+    - job: Wasm
```

Comment:
We need to put conditional inclusion tags around each of these jobs so that the jobs are only included for the platform selected when running the template